### PR TITLE
VZ-1315: zap structured logging

### PR DESCRIPTION
**Pull request by shihchang**
zap structured logging

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,17 +3,17 @@ module github.com/verrazzano/verrazzano-monitoring-operator
 go 1.13
 
 require (
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/gorilla/mux v1.7.3
 	github.com/kylelemons/godebug v1.1.0
 	github.com/prometheus/client_golang v1.2.1
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
```
Comment (desagar) [inline go.mod:9]: why did testify version change? I think it doesn't matter from an approval perspective since we only use it for tests, but curious why it changed

File: pkg/util/logs/logs.go [hunk 2]
```diff
@@ -4,41 +4,31 @@
...
-}
-
-// NewLogger creates a new log.Logger which sends logs to glog.Info.
-func NewLogger(prefix string) *log.Logger {
-	return log.New(GlogWriter{}, prefix, 0)
+// InitLogs initializes logs with Time and Global Level of Logs set at Info
+func InitLogs(opts kzap.Options) {
+	var config zap.Config
+	if opts.Development {
+		config = zap.NewDevelopmentConfig()
```
Comment (desagar) [inline pkg/util/logs/logs.go:16]: We are calling InitLogs with empty zap.Options{} - what is the default value for opts.Development then? Also, how is this related to the devProfile stuff used by the ES code in statefulset.go?

File: go.mod (hunk 1 above)
Comment (shihchang) [reply go.mod:9]: I saw verrazzano-operator is using v1.5.1, and it is approved, so i updated it.

File: pkg/util/logs/logs.go (hunk 2 above)
Comment (shihchang) [reply pkg/util/logs/logs.go:16]: opts.Development defaults to false. When opts.Development=true, it output to console instead of using json encoder

Comment (desagar) [reply pkg/util/logs/logs.go:16]: Also how do I set the option to something other than the default?

Comment (shihchang) [reply pkg/util/logs/logs.go:16]: verrazzano/install/chart/templates/03-verrazzano-monitoring-operator.yaml or edit deploy to add args ex.
```
args:
            - --zap-log-level=info
```
I will add this in my next PR on verrazzano/verrazzano

Comment (desagar) [reply pkg/util/logs/logs.go:16]: Ok thx - I would consider also adding the parameter for Development to false in the helm chart.

**Review by desagar — APPROVED** [review]
